examples/jfrog-platform/xray-with-pod-split/README.md

```diff
@@ -0,0 +1,180 @@
+# Deploying JFrog Xray
+
+To deploy JFrog Xray, you should create a new custom-values.yaml file with the following content and then pass it during installation with the Helm upgrade command.
+
+```yaml
+xray:
+  enabled: true
+```
+
+## Configuring JFrog Xray
+
+You can configure JFrog Xray with the following steps:
+
+1. View the main `values.yaml` file and copy the configurations that you wish to update into a `custom-values.yaml` file.
+2. Update the product configuration with custom values in the `custom-values.yaml` file. For the values not specified in the `custom-values.yaml` file, the default values from the main `values.yaml` of the chart will be used.
+3. Run the upgrade command.
+
+```console
+helm upgrade --install jfrog-platform --namespace jfrog-platform --create-namespace jfrog/jfrog-platform  -f custom-values.yaml
+```
+
+
+## Installing Xray
+
+### Pod Split ( Xray Version 3.124.x and above )
+
+- Architecture:
+
+<img src="./images/split-architecture.png" alt="Split Architecture" width="400">
+
+- All Xray services are running as separate containers in dedicated pods.
+
+#### Fresh Installation
+
+1.  Set the below parameters to true
+
+```bash
+xray:
+  enabled: true
+  splitXraytoSeparateDeployments:
+    fullSplit: true
+```
+2. Run the upgrade command.
+
+```console
+helm upgrade --install jfrog-platform --namespace jfrog-platform --create-namespace jfrog/jfrog-platform  -f custom-values.yaml
+```
+
+3. Please find the example custom values file here: [values-artifactory-xray.yaml](./values-artifactory-xray.yaml)
+
+
+#### Upgrading from Monolith to Pod Split
```

Comment:
Need a general explanation of the two step flow for a seamless upgrade. Before the examples and commands.